tests/test_click_odoo.py

```diff
@@ -235,6 +235,24 @@ def testcmd(env):
     assert "No database provided" in result.output
 
 
+def test_env_options_addons_path():
+    addons_path = ','.join([
+        os.path.join(odoo.__path__[0], 'addons'),
+        os.path.join(odoo.__path__[0], '..', 'addons'),
```

Comment:
@sbidoul Couldn't this line be also dropped?

Reply:
I think so, yes

Reply:
Ok, removed.